rules-reviewed/openshift/local-storage.windup.xml

```diff
@@ -9,7 +9,7 @@
             <addon id="org.jboss.windup.rules,windup-rules-javaee,2.7.0.Final" />
```

Comment:
this is a very minor issue almost non-issue ;) we don't update versions in existing rulesets.